Units/simple-nsis.d/expected.tags

```diff
@@ -0,0 +1,20 @@
+Answer	input.nsi	/^Var Answer$/;"	v
+UserName	input.nsi	/^Var UserName$/;"	v
+StartmenuFolder	input.nsi	/^Var StartmenuFolder$/;"	v
+UNINSTDIR	input.nsi	/^Var UNINSTDIR$/;"	v
+\x21Program Files	input.nsi	/^Section "!Program Files" SEC01$/;"	s
+Plugins	input.nsi	/^Section "Plugins" SEC02$/;"	s
+Language Files	input.nsi	/^Section "Language Files" SEC03$/;"	s
+Documentation	input.nsi	/^Section "Documentation" SEC04$/;"	s
+Autocompletion Tags	input.nsi	/^Section "Autocompletion Tags" SEC05$/;"	s
+GTK 	input.nsi	/^Section "GTK ${GTK_VERSION} Runtime Environment" SEC06$/;"	s
+Context Menus	input.nsi	/^Section "Context Menus" SEC07$/;"	s
+Desktop Shortcuts	input.nsi	/^Section "Desktop Shortcuts" SEC08$/;"	s
+Development files	input.nsi	/^Section "Development files" SEC09$/;"	s
+-AdditionalIcons	input.nsi	/^Section -AdditionalIcons$/;"	s
```

Comment:
The name started from - means "hidden section".
I wonder this should be part of the section name or not.
https://nsis.sourceforge.io/Reference/Section

Reply:
After thinking, I consider the characters (- and !) as  parts of sections.

sections are not identifiers. So none jumps to a section from an identifier. One may jump to the
section from an item in the navigation bar of one's editor or soemthing. It means strict name maching is not needed. Instead, informative tag names help users.